test/performance/broker-rmq/README.md

```diff
@@ -0,0 +1,78 @@
+kubectl apply --filename https://github.com/rabbitmq/messaging-topology-operator/releases/download/v1.2.1/messaging-topology-operator-with-certmanager.yaml
+# Installing Knative Eventing RabbitMQ Broker ...
+kubectl apply --filename https://github.com/knative-sandbox/eventing-rabbitmq/releases/download/v0.26.0/rabbitmq-broker.yaml
+```
+In case you want to test the latest dev version of eventing-rabbitmq, follow these steps:
+```
+# ⚠️ TODO after https://github.com/knative-sandbox/eventing-rabbitmq/pull/445 is merged:
+# git clone https://github.com/knative-sandbox/eventing-rabbitmq $GOPATH/src/knative.dev/eventing-rabbitmq
+git clone https://github.com/ikvmw/eventing-rabbitmq $GOPATH/src/knative.dev/eventing-rabbitmq
+cd $GOPATH/src/knative.dev/eventing-rabbitmq
+git checkout -b issue-239 origin/issue-239
+ko apply --filename config/broker
+```
+## 4/5. Run Knative Eventing RabbitMQ benchmarks
+```
+cd $GOPATH/src/knative.dev/eventing-rabbitmq
+ko apply --filename test/performance/broker-rmq/100-broker-perf-setup.yaml
+kubectl wait --for=condition=AllReplicasReady=true rmq/rabbitmq-test-cluster --timeout=10m --namespace perf-eventing
+kubectl wait --for=condition=IngressReady=true brokers/rabbitmq-test-broker --timeout=10m --namespace perf-eventing
+kubectl wait --for=condition=SubscriberResolved=true triggers/rabbitmq-broker-perf --timeout=10m --namespace perf-eventing
+ko apply --filename test/performance/broker-rmq/200-broker-perf.yaml
+```
+## 5/5. Download & visualise Knative Eventing RabbitMQ benchmark results
+Pre-requisite: [gnuplot](http://www.gnuplot.info/) (on macOS it's `brew install gnuplot`)
+```
+kubectl wait --for=condition=complete job/rabbitmq-broker-perf-send-receive --timeout=10m --namespace perf-eventing
+kubectl port-forward rabbitmq-broker-perf-aggregator 10001:10001 --namespace perf-eventing &
+PORT_FORWARD_PID=$!
+cd $GOPATH/src/knative.dev/eventing-rabbitmq/test/performance
+curl http://localhost:10001/results > eventing-rabbitmq-broker-perf-results.csv
+kill $PORT_FORWARD_PID
+wait $PORT_FORWARD_PID 2>/dev/null
+gnuplot -c latency-and-throughput.plg eventing-rabbitmq-broker-perf-results.csv 0.5 0 1100
+```
+![image](https://user-images.githubusercontent.com/90614196/136284992-b1ab58bd-17f5-4f01-bdb8-14655edad86e.png)
+To visualise just the end-to-end event latency, run: `gnuplot -c latency.plg eventing-rabbitmq-broker-perf-results.csv 0.5 0 1100`
+![image](https://user-images.githubusercontent.com/90614196/136285323-caa5dd6b-34fa-439e-9f7a-de4a9fd021ef.png)
+To visualise just the end-to-end event throughput, run: `gnuplot -c throughput.plg eventing-rabbitmq-broker-perf-results.csv 0.5 0 1100`
+![image](https://user-images.githubusercontent.com/90614196/136285415-676e8647-5a70-414c-a135-c9cf7795a103.png)
+> Notice in the screenshot above that the receiver througput line is around the zero axis, which makes me believe that we are just sending messages, but not receiving them. We should double-check.
```

Comment:
I saw the orange line at the 0 y value in my run as well. This should be a blocker.